### PR TITLE
iter8 kui integrations

### PR DESCRIPTION
**Pull request by sushmarchandran**
- Markdown for iter8-KUI integration

### CODE REVIEW

**Comment by sushmarchandran** [comment]
@Alan-Cha
1. Please let me know if I need to add a reference to the markdown anywhere else to view it on the website. 
2. Please also let me know if you are able to see the images on the markdown as expected.

**Comment by sushmarchandran** [comment]
@sriumcp @kalantar : Could you review the content of the markdown and made edits as necessary?

File: content/integrations/kui.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+date: 2020-08-06T12:00:00+00:00
+menuTitle: Kui
+title: Integrating Iter8 with KUI
+weight: 10
+summary: Describes iter8's integrations with KUI
+---
+
+[Kui](https://kui.tools) combines the power of familiar CLIs with visualizations in high-impact areas. Kui enables you to manipulate complex JSON and YAML data models, integrate disparate tooling, and provides quick access to aggregate views of operational data.
+
+The iter8-KUI integration enables user-intuitive and interactive options to use iter8. Using KUI, you can run Human-In-The-Loop iter8 experiments and easily modifiable and customizable metrics as you will see below.
```
Comment (kalantar) [inline content/integrations/kui.md:12]: modifiable --> modify
customizable --> customize

**Review by kalantar — DISMISSED** [review]
Just a couple of typos I think.

**Review by Alan-Cha — DISMISSED** [review]

**Review by Alan-Cha — APPROVED** [review]

**Pull request merged**